src/CMakeLists.txt

```diff
@@ -117,19 +117,20 @@ if(WIN32 OR APPLE)
 endif()
 
 if(BUILD_FAST_TOKENIZERS)
-  # FastTokenizers use static MSVC runtime, hence we have to use it as well
-  set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
   set(THIRD_PARTY_BUILD_TYPE ${CMAKE_BUILD_TYPE})
-
+  # Set FastTokenizers to use dynamic MSVC runtime
+  set(MSVC_STATIC_CRT OFF)  # PADDLE_LIB
+  set(PCRE2_STATIC_RUNTIME OFF) # PCRE2_LIB
+  set(SPM_ENABLE_MSVC_MT_BUILD OFF) # sentencepiece libs
   if(WIN32)
     foreach(
         flag_var
-        CMAKE_CXX_FLAGS_DEBUG
-        CMAKE_C_FLAGS_DEBUG)
-        if(${flag_var} MATCHES "/MD")
-          string(REGEX REPLACE "/MD" "/MTd" ${flag_var} "${${flag_var}}")
-        elseif(NOT (${flag_var} MATCHES "/MTd"))
-          set(${flag_var} "${${flag_var}} /MTd")
+        CMAKE_CXX_FLAGS
+        CMAKE_C_FLAGS)
+        if(${flag_var} MATCHES "/MT")
+          string(REGEX REPLACE "/MT" "/MD$<$<CONFIG:Debug>:d>" ${flag_var} "${${flag_var}}")
+        elseif(NOT (${flag_var} MATCHES "/MD$<$<CONFIG:Debug>:d>"))
+          set(${flag_var} "${${flag_var}} /MD$<$<CONFIG:Debug>:d>")
```

Comment:
If that relates to that assertion error at runtime for debug at add_extension(), I tried compiling with `-DBUILD_FAST_TOKENIZERS=OFF` earlier and it didn't resolve the issue.

Reply:
Anyway, changes are useful 😄